testing/btest/broker/store/brokerstore-backend-sqlite-corrupt-delete.zeek

```diff
@@ -0,0 +1,42 @@
+# @TEST-DOC: Populate a database, corrupt it then observe Zeek's behavior deleting the database and reopening it.
+# @TEST-REQUIRES: dd --version > dd-version.txt
```

Comment:
Just curious here whether we need to keep the version. Fine with me if you feel it's useful.